openssl/src/x509/mod.rs

```diff
@@ -548,6 +589,20 @@ impl X509Ref {
         }
     }
 
+    /// Returns certificate version. If this certificate has no explicit version set, it defaults to
+    /// version&nbsp;1.
+    ///
+    /// Note that `0` return value stands for version 1, `1`&nbsp;&ndash; for version 2 and so on.
+    ///
+    /// This corresponds to [`X509_get_version`].
+    ///
+    /// [`X509_get_version`]: https://www.openssl.org/docs/man1.1.1/man3/X509_get_version.html
+    pub fn version(&self) -> Result<i64, ErrorStack> {
+        // Covered with `x509_ref_version()`, `x509_ref_version_no_version_set()`,
+        // `x509_ref_version_incorrect_version_set()` tests
+        unsafe { cvt_long_n(ffi::X509_get_version(self.as_ptr())) }
```

Comment:
The man page for X509_get_version does not claim it ever returns an error.

Reply:
Removed error check.